hypervisor/devicemap.go

```diff
@@ -506,6 +524,19 @@ func (ctx *VmContext) removeInterface() {
 	}
 }
 
+func (ctx *VmContext) removeInterfaceByLinkIndex(index int) {
+	for idx, nic := range ctx.devices.networkMap {
+
+		glog.V(1).Infof("==============>idx:nic is %v:%v", idx, nic)
```

Comment:
This is used to see what we have in the networkMap, it's a debug print, will remove in the future.